### PR TITLE
Change some suit storage sprites to be on the hip instead of the back

### PR DESCRIPTION
**Pull request by Dutch-VanDerLinde**
Sprites are from Cm13

They look better with backpacks, the gun back sprites are supposed to be if you're wearing them on your back, the suit slot ones look like they're on the hip

### CODE REVIEW

**Review by DrSmugleaf — DISMISSED** [review]
Thanks for the PR!

File: Resources/Textures/_RMC14/Objects/Weapons/Guns/Rifles/m54c.rsi/equipped-SUITSTORAGE.png
Comment (QuietlyWhisper) [inline Resources/Textures/_RMC14/Objects/Weapons/Guns/Rifles/m54c.rsi/equipped-SUITSTORAGE.png:1]: This sprite cannot be used

File: Resources/Textures/_RMC14/Objects/Weapons/Guns/Rifles/m54ce2.rsi/equipped-SUITSTORAGE.png
Comment (QuietlyWhisper) [inline Resources/Textures/_RMC14/Objects/Weapons/Guns/Rifles/m54ce2.rsi/equipped-SUITSTORAGE.png:1]: This sprite cannot be used

**Review by QuietlyWhisper — CHANGES_REQUESTED** [review]

**Comment by Dutch-VanDerLinde** [comment]
Changed the M54CE2 and M54C suit slots to use a modified version of the M4SPR suit slot rather than M45A1 sprites

**Review by QuietlyWhisper — APPROVED** [review]

**Pull request merged**